src/main/java/com/google/sps/servlets/EventServlet.java

```diff
@@ -60,7 +60,23 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
       getEvents(request, response, userService);
       return;
     }
-    // TODO: parse "events/{event_id}" here.
+    
+    String[] pathParts = pathName.split("/");
+    String eventId = pathParts[1];
+
+    if (pathParts[2].equals("join")) {
```

Comment:
Maybe later we could find a better way to do these checks so it's the same for gcalendar, search, etc.